### ISSUE TITLE
Not working in 1.9.4?

### ISSUE THREAD

**Issue by cj89898** [McJty/CombatHelp#3]
The shield is not equipping when using 1.9.4 version of minecraft

Neither is storing things.

**Comment by McJty** [comment]
Yes, I have found the same issue in my lets play. In dev env it works ok. I have to investigate what is going on there

**Comment by cj89898** [comment]
Thanks, I know you're quite a busy guy!
